lib/sqewer.rb

```diff
@@ -1,3 +1,5 @@
+require 'connection_pool'
+
```

Comment:
We do not need to require `connection_pool` or depend on it - we can provide a `NullPool` instead (Prorate gem has a similar setup). Less dependencies that way :-) and if we document connection_pool compatibility people may install it themselves